lib/OpenQA/Worker/Jobs.pm

```diff
@@ -717,6 +717,8 @@ sub upload_status {
     }
 
     # if there is nothing to say, don't say it (said my mother)
+    # FIXME: When would this block ever be executed?
```

Comment:
Indeed I think this is dead code...

Reply:
This was refactored - it was meaningful in 5ee879d77f402f4bdb95d4cb5ab3e1a3566b9886 - even though I don't remember my mother saying that :)